app/views/layouts/_footer.html.slim

```diff
@@ -0,0 +1,36 @@
+footer class="govuk-footer no-print" role="contentinfo"
```

Comment:
The shorthand for classes in slim, which I think is more idiomatic, is to use `.` syntax. So this would be: `footer.govuk-footer.no-print role="contentinfo"`

Reply:
thanks, this is due to my unfamiliarity so all pointers are welcome